README.md

```diff
@@ -9,16 +9,19 @@ it is still in very early stage, but you can try use it like this:
 var RequirejsPluginsLoaderPlugin = require('require-js-plugins-loader/Plugin');
 
 module.exports = {
-  entry: "./index.js",
+  entry: './index.js',
   output: {
     path: __dirname,
-    filename: "bundle.js"
+    filename: 'bundle.js'
   },
   module: {
     loaders: [
       { test: /\.js$/, loader: 'require-js-plugins' }
     ]
   },
+  requirejsPlugins: {
```

Comment:
Is this required configuration now? If no, are we parsing all something!args and assuming that they should be acted on?